metals/src/main/scala/scala/meta/internal/metals/UserConfiguration.scala

```diff
@@ -205,6 +215,28 @@ object UserConfiguration {
             Some(value)
         }
       }
+
+    def getStringListKey(key: String): Option[List[String]] =
+      getKey[List[String]](
+        key,
+        { elem =>
+          if (elem.isJsonArray()) {
+            val parsed = elem.getAsJsonArray().asScala.flatMap { value =>
+              Try(value.getAsJsonPrimitive().getAsString()) match {
+                case Failure(exception) =>
+                  errors += s"json error: values in '$key' should have value of type string but obtained $value"
+                  None
+                case Success(value) =>
+                  Some(value)
+              }
+            }
+            Option(parsed.toList)
```

Comment:
you can do `Some(...)` (it is impossible to be null here.

Reply:
fixed!